src/components/Watch.svelte

```diff
@@ -86,6 +92,14 @@
     });
   };
   $: setTimeout(() => changeSide($videoSide, $chatSplit), 0);
+
+  function orientationChangeCallback() {
+    $videoSide = ($autoVertical && window.innerHeight > window.innerWidth) ? VideoSide.TOP : $videoSideSetting;
+  }
+  window.addEventListener('resize', orientationChangeCallback);
+  videoSideSetting.subscribe(orientationChangeCallback);
+  autoVertical.subscribe(orientationChangeCallback);
```

Comment:
Should unsubscribe at onDestroy.

Or even better, replace it with `$: $videoSideSetting, $autoVertical, orientationChangeCallback();`, which would call the function whenever any of the auto-subscribe stores updates.

Reply:
You could also make a ``changeOrientation`` function and do
```js
$: changeOrientation($videoSideSetting, $autoVertical)
```

Then you can call this function in the orientationChangeCallback